exercises/run-length-encoding/src/test/java/RunLengthEncodingTest.java

```diff
@@ -0,0 +1,79 @@
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+public class RunLengthEncodingTest {
+    private RunLengthEncoding runLengthEncoding;
+
+    @Before
+    public void setUp() {
+        runLengthEncoding = new RunLengthEncoding();
+    }
+
+    @Test
+    public void encodeEmpty() {
+        Assert.assertEquals("", runLengthEncoding.encode(""));
+    }
+
```

Comment:
Are you sure?  Other tests I have looked at leave the `@Ignore` off of the first/simplest test.